util.sh

```diff
@@ -123,7 +123,12 @@ fats_setup_gcloud() {
   sudo apt-get update && sudo apt-get install google-cloud-sdk
 
   gcloud config set project cf-spring-pfs-eng
+  gcloud config set compute/region us-central1
   gcloud config set compute/zone us-central1-a
+  # TODO debug why config set for region and zone does not work
+  # current workaround is to use the env variables below
+  export CLOUDSDK_COMPUTE_REGION="us-central1"
```

Comment:
we should be able to lookup these value with gcloud when we need them, `gcloud config get-value compute-region`